first_unix_commands.sh

```diff
@@ -0,0 +1,13 @@
+echo "Samah" > myname.txt
+touch myname.txt
+echo "Samah" >> myname.txt
```

Comment:
You are effectively writing to the file in the same way. What other ways are there to write to a file?